controllers/filterable-list.js

```diff
@@ -1,8 +1,6 @@
-var edit = require('../services/edit'),
-  dom = require('@nymag/dom'),
+const dom = require('@nymag/dom'),
```

Comment:
using const for importing dependencies makes sense but it's inconsistent across the rest of the project. no big, but worth considering when what we're looking for during code review, etc. if we're going to start going with `const`s then we should replicate/enforce that elsewhere in kiln code.